### ISSUE TITLE
ERROR in Value at position 12 in the NgModule.declarationss of NgStringPipesModule is not a reference

### ISSUE THREAD

**Issue by ciukstar** [danrevah/ngx-pipes#151]
**I'm submitting a ...**  (check one with "x")
```
[ x] bug report => search github for a similar issue or PR before submitting
[ ] feature request
```

* **Ngx-pipes version:** `ngx-pipes@2.5.1`

**Current behavior**
Does not compile with `"enableIvy": true`
`ERROR in Value at position 12 in the NgModule.declarationss of NgStringPipesModule is not a reference: [object Object]`

**Expected behavior**
Does copile with `"enableIvy": true`

**Minimal reproduction of the problem with instructions**
`$ cat tsconfig.json`:
```
{
  "compileOnSave": false,
  "compilerOptions": { ... },
  "angularCompilerOptions": {
    "enableIvy": true
  }
}
```

**Please tell us about your environment:**

* **Angular version:**
```
$ ng version
Angular CLI: 8.0.3
Node: 8.10.0
OS: linux x64
Angular: 8.0.1
... animations, cdk, common, compiler, compiler-cli, core, forms
... language-service, material, platform-browser
... platform-browser-dynamic, router

Package                           Version
-----------------------------------------------------------
@angular-devkit/architect         0.800.3
@angular-devkit/build-angular     0.800.3
@angular-devkit/build-optimizer   0.800.3
@angular-devkit/build-webpack     0.800.3
@angular-devkit/core              8.0.3
@angular-devkit/schematics        8.0.3
@angular/cli                      8.0.3
@angular/flex-layout              8.0.0-beta.26
@ngtools/webpack                  8.0.3
@schematics/angular               8.0.3
@schematics/update                0.800.3
rxjs                              6.5.2
typescript                        3.4.5
webpack                           4.30.0
```

* **Node (for AoT issues):** `node --version` =   v8.10.0

**Comment by danrevah** [comment]
try the solution in #150 
did it help?

**Comment by danrevah** [comment]
This might do the trick: https://github.com/danrevah/ngx-pipes/issues/150#issuecomment-497964650

But, this needs to be fixed for es5 as well. 

So any fixing PR is more than welcome :)

**Comment by elementalPRESS** [comment]
> This might do the trick: [#150 (comment)](https://github.com/danrevah/ngx-pipes/issues/150#issuecomment-497964650)
> 
> But, this needs to be fixed for es5 as well.
> 
> PR is more than welcome :)

This doesn't work for me :-(

**Comment by danrevah** [comment]
https://github.com/danrevah/ngx-pipes/issues/150#issuecomment-509029926

A fix was released in v2.5.5
